torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -39,25 +39,25 @@ using namespace torch::autograd::generated;
 
 namespace torch { namespace autograd {
 
-inline void check_inplace(const Tensor& tensor) {
+inline void check_inplace(const Tensor& tensor, bool requires_grad) {
```

Comment:
It would be helpful for future readers to explain here why `requires_grad` is different from `tensor.requires_grad()` (this is clear contextually in the PR but not obvious at this function)